crates/bevy_math/src/bounding/raytest3d.rs

```diff
@@ -0,0 +1,85 @@
+use super::{Aabb3d, BoundingSphere, IntersectsVolume};
+use crate::{primitives::Direction3d, Vec3};
+
+/// A raycast intersection test for 3D bounding volumes
+pub struct RayTest3d {
+    /// The origin of the ray
+    pub origin: Vec3,
+    /// The direction of the ray
+    pub dir: Direction3d,
+    /// The inverse direction of the ray
```

Comment:
It's the inverse for each component, it's the same as calling `dir.recip()`. I've noticed more people stumble on this, it might make sense to rename it to `dir_recip` or something else that at least encourages people to check the comment (which hopefully better explains what it is and why it's useful)